crates/subspace-node/src/chain_spec.rs

```diff
@@ -71,20 +77,13 @@ pub fn get_account_id_from_seed(seed: &str) -> AccountId {
     AccountPublic::from(get_from_seed::<sr25519::Public>(seed)).into_account()
 }
 
-#[cfg(feature = "json-chain-spec")]
-pub fn subspace_testnet_config() -> Result<SubspaceChainSpec, String> {
-    SubspaceChainSpec::from_json_bytes(&include_bytes!("../../../chain-spec.json")[..])
+pub fn testnet_config_json() -> Result<SubspaceChainSpec, String> {
```

Comment:
I'm wondering if it's better to use the publicly announced network name given that it's there already, all the configs with development purpose don't have a dedicated name. `testnet` and `testnet-compiled` is not super distinctive to me, `aries` seems to also bring in some ceremony.

```suggestion
pub fn aries_config() -> Result<SubspaceChainSpec, String> {
```

Reply:
I don't think we used aries anywhere in the codebase yet. It is just on the website and in some other places.

Reply:
Well, worth starting using it from somewhere if you think it's a good direction, otherwise I'm fine with it as is.